packages/react-resizable-panels/package.json

```diff
@@ -71,19 +47,21 @@
     "clear:builds": "rm -rf ./packages/*/dist",
     "clear:node_modules": "rm -rf ./node_modules",
     "lint": "eslint \"src/**/*.{ts,tsx}\"",
-    "test": "jest --config=jest.config.js",
-    "test:watch": "jest --config=jest.config.js --watch",
+    "test:browser": "vitest",
+    "test:node": "vitest -c vitest.node.config.ts",
     "watch": "parcel watch --port=2345"
   },
   "devDependencies": {
     "@babel/plugin-proposal-nullish-coalescing-operator": "7.18.6",
     "@babel/plugin-proposal-optional-chaining": "7.21.0",
+    "@vitest/ui": "^3.1.2",
     "eslint": "^8.37.0",
     "eslint-plugin-react-hooks": "^4.6.0",
-    "jest": "^29.7.0",
-    "jest-environment-jsdom": "^29.7.0",
+    "jsdom": "^26.1.0",
     "react": "experimental",
-    "react-dom": "experimental"
+    "react-dom": "experimental",
+    "save-dev": "0.0.1-security",
```

Comment:
Is this intentional...?

Reply:
No, it was accidentally introduced in https://github.com/bvaughn/react-resizable-panels/pull/476/files#diff-b9daed64243c63f6aac6e93e6d75abb73e84a246c863e14171ae0ba33775aaf7R63 but, luckily, this isn't a harmful package.

Reply:
No. This is a result of me trying to maintain this stuff after hours when I'm half awake.